subprojects/errorprone/src/test/java/org/mockito/errorprone/bugpatterns/MockitoMockedStaticTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) 2017 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.errorprone.bugpatterns;
+
+import com.google.errorprone.CompilationTestHelper;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+@RunWith(JUnit4.class)
+public class MockitoMockedStaticTest {
+
+    private CompilationTestHelper compilationTestHelper;
+
+    @Before
+    public void setup() {
+        compilationTestHelper =
+            CompilationTestHelper.newInstance(MockitoMockedStatic.class, getClass());
+    }
+
+    @Test
+    public void mockedStaticWithMockField_shouldError() {
+        compilationTestHelper
+            .addSourceLines(
+                "input.java",
+                "import org.mockito.Mock;",
+                "import org.mockito.MockedStatic;",
+                "class Test {",
+                "  @Mock",
+                "  private Object mock;",
+                "  public void test() {",
+                "    try (MockedStatic<Object> mockedStatic = Mockito.mockStatic(Object.class)) {",
+                "      // Do something with the mock",
+                "    }",
+                "  }",
+                "}")
+            .doTest();
+    }
+
+    @Test
+    public void mockedStaticWithMockParameter_shouldError() {
+        compilationTestHelper
+            .addSourceLines(
+                "input.java",
+                "import org.mockito.Mock;",
+                "import org.mockito.MockedStatic;",
+                "class Test {",
```

Comment:
In the case of the issue, this was when marking a field as a mock rather than a parameter itself. Is it worth including a test for that case as well perhaps?

Reply:
Sorry if I didn't do the test correctly, I made the mockedStaticWithMockField_shouldError test. Is that something different to what you are talking about?

Reply:
Ahhhh ignore me. I was reading this on mobile and totally missed the other method! Yeah that looks right to me!

Sorry about the confusion